environments/hca-prod/terraform/buckets.tf

```diff
@@ -83,16 +82,28 @@ resource google_storage_bucket ucsc_bucket {
   location = "US"
 }
 
-resource google_storage_bucket_iam_member ucsc_bucket_iam {
+resource google_storage_bucket_iam_member ucsc_bucket_reader_iam {
   provider = google-beta.target
   bucket   = google_storage_bucket.ucsc_bucket.name
   # When the storage.admin role is applied to an individual bucket,
   # the control applies only to the specified bucket and objects within
   # the bucket: https://cloud.google.com/storage/docs/access-control/iam-roles
-  for_each = toset(["hannes@ucsc.edu", "dsotirho@ucsc.edu"])
+  for_each = toset(["platform-hca-dev-read@ucsc.edu", "platform-hca-prod-read@ucsc.edu"])
```

Comment:
❓ Why is the `platform-hca-dev-read@ucsc.edu` user getting reader access to a prod bucket?